examples/lock-app/k32w/README.md

```diff
@@ -175,6 +175,9 @@ Also, in case the OM15082 Expansion Board is not attached to the DK6 board, the
 build argument (chip_with_OM15082) inside the gn build instruction should be set
 to zero. The argument chip_with_OM15082 is set to zero by default.
 
+In case that Openthread CLI is needed, chip_with_ot_cli build argument must be
+set to 1.
```

Comment:
since this is `gn gen` could we use booleans instead of 1/0?

I think this would apply to both chip_with_OM15082 and chip_with_ot_cli. I see is_debug taking "false" instead of 0. please open a separate issue for this.

Reply:
Thanks @andy31415 I will submit a separate PR for that